services/core-api/app/api/now_applications/resources/now_application_status_resource.py

```diff
@@ -141,6 +143,17 @@ def put(self, application_guid):
 
                 permit_amendment.save()
 
+                #Issue Permit as Verifiable Credential to OrgBook
+                try:
+                    OrgBookIssuerService().issue_permit_amendment_vc(permit_amendment)
```

Comment:
Do we want to check the return value of this here, i.e. the `response.status_code` maybe?

Reply:
I don't want this code to effect this method at all. I could be talked into not modifying this at all.

Reply:
Up to you guys on this one.